src/media/css/feed/collections.styl

```diff
@@ -55,7 +55,14 @@
 .feed-landing-header-mobile .feed-collection {
     display: block;
     margin: 0 auto;
-    width: $mobile-content-width;
+}
+
+.feed-landing-header-mobile .feed-item {
```

Comment:
We can't do this because the tiles don't stretch

![screen shot 2015-02-23 at 1 26 55 pm](https://cloud.githubusercontent.com/assets/674727/6337619/aa8675e4-bb5f-11e4-9ae8-de2d2b1a1ee0.png)